docs/intro/tutorial04.txt

```diff
@@ -20,16 +20,16 @@ tutorial, so that the template contains an HTML ``<form>`` element:
 .. code-block:: html+django
     :caption: polls/templates/polls/detail.html
 
-    <h1>{{ question.question_text }}</h1>
-
-    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
-
     <form action="{% url 'polls:vote' question.id %}" method="post">
     {% csrf_token %}
+    <fieldset>
+    <legend>{{ question.question_text }}</legend>
+    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
```

Comment:
Reviewing https://design-system.service.gov.uk/components/radios/, which I often use as a reference implementation, it looks like we could do even better in this example:

```suggestion
    <legend><h1>{{ question.question_text }}</h1></legend>
    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
```

Then the heading is present, and there being an `<h1>` in the legend doesn’t prevent the `fieldset` group being labelled correctly (tested in VoiceOver + Safari on https://design-system.service.gov.uk/components/radios/hint/index.html).